cmstestsuite/Tests.py

```diff
@@ -53,18 +53,32 @@
 LANG_PYPY3 = "Python 3 / PyPy"
 LANG_RUST = "Rust"
 LANG_C_SHARP = "C# / Mono"
+
 ALL_LANGUAGES = (
-    LANG_CPP, LANG_CPP14, LANG_CPP17, LANG_CPP20, LANG_C, LANG_HS, LANG_JAVA, LANG_PASCAL,
-    LANG_PHP, LANG_PYTHON3, LANG_PYPY3, LANG_RUST, LANG_C_SHARP
-)
-NON_INTERPRETED_LANGUAGES = (
-    LANG_C, LANG_CPP, LANG_CPP14, LANG_CPP17, LANG_CPP20, LANG_PASCAL
-)
-COMPILED_LANGUAGES = (
-    LANG_C, LANG_CPP, LANG_CPP14, LANG_CPP17, LANG_CPP20, LANG_PASCAL, LANG_JAVA,
-    LANG_PYTHON3, LANG_PYPY3, LANG_HS, LANG_RUST, LANG_C_SHARP
+    LANG_C,
+    LANG_C_SHARP,
+    LANG_CPP, LANG_CPP14, LANG_CPP17, LANG_CPP20,
+    LANG_HS,
+    LANG_JAVA,
+    LANG_PASCAL,
+    LANG_PHP,
+    LANG_PYTHON3, LANG_PYPY3,
+    LANG_RUST,
 )
 
+# Languages which support compilation with a manager/grader.
+# TODO: this should be the same as `ALL_LANGUAGES`.
+MANAGER_LANGUAGES = [
+    lang for lang in ALL_LANGUAGES if lang not in (LANG_HS, LANG_PHP, LANG_RUST)
+]
+
+# Languages for which solutions to be linked to a stub for communication tasks is
```

Comment:
nit: `should` be linked?

Reply:
No, this is correct (except I should have written `are` instead of `is`, fixed)